Mathlib/Topology/DiscreteQuotient.lean

```diff
@@ -0,0 +1,423 @@
+/-
+Copyright (c) 2021 Adam Topaz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Calle Sönne, Adam Topaz
+
+! This file was ported from Lean 3 source module topology.discrete_quotient
+! leanprover-community/mathlib commit d101e93197bb5f6ea89bd7ba386b7f7dff1f3903
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Topology.Separation
+import Mathlib.Topology.SubsetProperties
+import Mathlib.Topology.LocallyConstant.Basic
+
+/-!
+
+# Discrete quotients of a topological space.
+
+This file defines the type of discrete quotients of a topological space,
+denoted `discrete_quotient X`. To avoid quantifying over types, we model such
```

Comment:
```suggestion
denoted `DiscreteQuotient X`. To avoid quantifying over types, we model such
```
Here and in the rest of the module docstring

Reply:
Sorry, I forgot to run `fix-comments`